### ISSUE TITLE
have a greyscale representation

### ISSUE THREAD

**Issue by theod** [ossia/score#141]
this is not for the next release but @cbossut said we should be able to display a score in greyscale to increase accessibility.

**Comment by nvuaille** [comment]
When compiling there's a CMAKE option that set colors in grayscale. (it's called something like ISCORE_IEEE_SKIN ).

**Comment by jcelerier** [comment]
indeed, running CMake with `-DISCORE_IEEE:Bool=True` (or setting it up in the gui) will put a black-on-white color scheme suitable for publications, along with more "readable" (but less pretty) fonts.

**Comment by cbossut** [comment]
The point was especially to say that we should not use only color to show
something in the GUI because this information could be lost in gray-scale.

2016-02-04 12:59 GMT+01:00 Jean-Michaël Celerier notifications@github.com:

> indeed, running CMake with -DISCORE_IEEE:Bool=True (or setting it up in
> the gui) will put a black-on-white color scheme suitable for publications,
> along with more "readable" (but less pretty) fonts.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/OSSIA/i-score/issues/141#issuecomment-179790050.

## 

Clément Bossut

**Comment by jcelerier** [comment]
the skin can be changed in the settings for what it's worth

**Comment by bltzr** [comment]
so we can close this issue now, right ?

**Comment by jcelerier** [comment]
if it's enough, yes.

**Comment by bltzr** [comment]
Well as I don't care about this, this is enough for me... So I'm closing the issue. If that's very important for someone, they should reopen it and set its milestone to 1.5
